src/python/visclaw/data.py

```diff
@@ -621,12 +622,17 @@ def __init__(self, name, figno, fig_type, plotdata):
         self._show = True
         self.add_attribute('plotaxes_dict', {})
         self.add_attribute('type',fig_type)   # = 'each_frame' or 'each_run' or 'each_gauge'
+        self.add_attribute('use_for_kml',False)
+        self.add_attribute('kml_dpi',200)
```

Comment:
DPI is usually set as a dictionary key in the `kwargs` dictionary.  It might be worthwhile to check to see if it has been specified there.